From: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
Subject: Fix `gtk-update-icon-cache` call
Signed-off-by: Mattéo Rossillol‑‑Laruelle <beatussum@protonmail.com>
Description:
Call `gtk-update-icon-cache` during `pkg_postrm()` instead of during
`src_compile()`.

--- a/install.sh
+++ b/install.sh
@@ -107,8 +107,6 @@ install_theme() {
   ln -sr "${THEME_DIR}/24"                                                       "${THEME_DIR}/24@2x"
   ln -sr "${THEME_DIR}/32"                                                       "${THEME_DIR}/32@2x"
   ln -sr "${THEME_DIR}/scalable"                                                 "${THEME_DIR}/scalable@2x"
-
-  gtk-update-icon-cache "${THEME_DIR}"
 }

 while [ $# -gt 0 ]; do
